table/block_based/block_based_table_factory.cc

```diff
@@ -310,6 +309,10 @@ static std::unordered_map<std::string, OptionTypeInfo>
          {offsetof(struct BlockBasedTableOptions, optimize_filters_for_memory),
           OptionType::kBoolean, OptionVerificationType::kNormal,
           OptionTypeFlags::kNone}},
+        {"use_delta_encoding",
```

Comment:
This is a pretty important option. It controls encoding of newly generated files and also decoding of existing sst files. I wasn't aware that this option was not included in the OPTIONS file before. I'm not sure about the impact of starting to persist this option and load it from OPTIONS file. Let's not include this change in the PR.

Reply:
We should update the test if we ever fix this: https://github.com/facebook/rocksdb/blame/40b618f2349b509eabdd175f75faf7ce84cf0696/options/options_settable_test.cc#L160-L161

Reply:
I pushed a commit to back this change out